Bottom: b274849078f71405b6bcf4ccc1ee3768bccc3546
Top:    06b045eaee28f68625e1725b496d62d3f1888bf6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:59:30 +0000

Refresh of add-foundcolor-to-resource-set

---

diff --git a/src/main.c b/src/main.c
index e41fc55..d80612d 100644
--- a/src/main.c
+++ b/src/main.c
@@ -770,11 +770,11 @@ Color to indicate physical connections. Default: @samp{#00ff00}
 /* %start-doc options "3 Colors"
 @ftable @code
 @item --found-color <string>
-Color to indicate logical connections. Default: @samp{#00ff00}
+Color to indicate logical connections. Default: @samp{#ff00ff}
 @end ftable
 %end-doc
 */
-  COLOR (FoundColor, "#00ff00", "found-color",
+  COLOR (FoundColor, "#ff00ff", "found-color",
 	 "color to indicate logically connected objects"),
 
 /* %start-doc options "3 Colors"
